express-rest-api/server/app.js

```diff
@@ -0,0 +1,27 @@
+var createError = require('http-errors');
+var express = require('express');
+var path = require('path');
+
+var indexRouter = require('./routes/index');
+var usersRouter = require('./routes/users');
+
+var app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+app.use('/', indexRouter);
+app.use('/users', usersRouter);
+
+// catch 404 and forward to error handler
```

Comment:
`// catch undefined routes and respond with 404`

Reply:
Oh this comment is much easier to understand! Thanks!!!